components/Header.tsx

```diff
@@ -0,0 +1,18 @@
+import { Image } from "@chakra-ui/image";
+import { Flex } from "@chakra-ui/layout";
```

Comment:
_(arbitrary comment location to start a thread)_

I'm getting tons of 429's in the console because we use a default provider that's getting rate-limited. Can we pull the RPC info out into an environment variable?

Reply:
We have the Alchemy provider setup. If you copy the .env.sample and fill in the API_KEY env var with an alchemy token those should go away. I can probably update the README to be clearer.

Reply:
Ah yea I didn't realize we needed those env vars, the names seem generic so I thought they were defaults from scaffolding. Let's rename to `NEXT_PUBLIC_ALCHEMY_API_KEY` to be more specific

And wdyt about deleting `NEXT_PUBLIC_ENABLE_TESTNETS` and instead doing `...(process.env.NODE_ENV === 'development' ? [goerli] : [])`?

Reply:
> And wdyt about deleting NEXT_PUBLIC_ENABLE_TESTNETS and instead doing ...(process.env.NODE_ENV === 'development' ? [goerli] : [])?

I don't think we should do this, actually. Supporting Goerli (on the production deploy) is one of the requirements.

Reply:
Removed testnet check

Reply:
ah good call @apbendi so yea we don't any conditionals, marking this resolved